### ISSUE TITLE
[BUG] [Linter] Tag should be self closing. <meta (H017)

### ISSUE THREAD

**Issue by alessandro-newzoo** [Riverside-Healthcare/djLint#473]
- [x] I'm on the [latest version](https://pypi.org/project/djlint/) of djLint
- [x] I've searched the [issues](https://github.com/Riverside-Healthcare/djLint/issues)
- [x] I've read the [docs](https://djlint.com)

## System Info
 - OS: Linux 8251ef520be7 5.4.0-131-generic #147-Ubuntu SMP Fri Oct 14 17:07:22 UTC 2022 x86_64 GNU/Linux
 - Python Version 3.9.2
 - djLint Version 1.19.7
 - template language: twig

## Issue

`<meta>` tags should not be self-closed but they're still being reported as a problem.

## How To Reproduce
**test.twig:**
```html
<meta charset="UTF-8">
```

**Error:**
```
Tag should be self closing. <meta charset=\"UTF-8 (H017)",
```

## Contents of .djlintrc/pyproject.toml [tool.djlint]
```
{}
```

Thank you so much for the only working twig linter/formatter out there ❤️

**Comment by christopherpickering** [comment]
hey @alessandro-newzoo thanks :)

for the `<meta` and any self closing tags, it is purely a preference. I know they say that `meta` is different than a `<img` tag for example, but on modern html, you can leave out the `/` or add it as you prefer. In your case I would just disable h017.

Or do you have a different understanding of this? Were you execting H017 to be reported for `<img >` but not `<meta >`? 

Thanks!

**Comment by alessandro-newzoo** [comment]
Hi @christopherpickering thanks for your reply!

> Or do you have a different understanding of this? Were you execting H017 to be reported for <img > but not <meta >?

Yes that's what I meant :)
But after researching more about the topic, I just realised that, like you said, with HTML5 it's fine to never self-close anything, so I'm just gonna do that moving forward and ignore `H017` :)

Still, it's _technically_ wrong for djlint to report that `<meta>` should be self closing - But that's fine with me 😄

Thanks again man!

**Comment by christopherpickering** [comment]
😁 self closing is an argument with out an end 😀 

Thank you! I saw the feature request on the vscode repo, you can open it here if you'd like. I think there may actually be an issue for it - you can append your examples to it maybe.

**Comment by alessandro-newzoo** [comment]
Do you mean this one? https://github.com/monosans/djlint-vscode/issues/115#issuecomment-1337660037

I tried looking for similar issues here, but the only one I could find relates to H017: https://github.com/Riverside-Healthcare/djLint/issues/252

My issue is related to T002 and T003, so I'm gonna open a new one :)

Thanks again!
